api/src/main/resources/workbench-api.yaml

```diff
@@ -5000,12 +5000,9 @@ definitions:
         default: false
       kernelType:
         "$ref": "#/definitions/KernelTypeEnum"
-      genomicsDataType:
-        type: string
-        default: NONE
-        enum:
-          - NONE
-          - WHOLE_GENOME
+      generateGenomicsAnalysisCode:
```

Comment:
I think the previous structure was more extensible. This is a field which could potentially need more than two values, which is a sign you should use an enum rather than a boolean. It is possible we could add simple codegen for microarray in the future

Reply:
I actually wasn't thinking of this as replacing `genomicsDataType` but more so that `generateGenomicsAnalysisCode` is useful in its own right for indicating the user's intent on what they would like to be generated as part of the export request.

I think `genomicsDataType` can be brought back along with `generateGenomicsAnalysisCode` once we get to the point where we want to support both WGS and microarray.

Also, it's also possible that this field is redundant with what's in the referenced `dataset` if we're just using it as a field to describe what's in the dataset. For example, it's possible to inspect the dataset model to determine if it contains WGS or not. I could potentially see this being the case for microarray data as well.

Reply:
OK, fine leaving this as-is if we may scrap the field entirely anyways.